### PR TITLE
Chore/integrate babel and scss

### PR DESCRIPTION
**Pull request by peygar**
- es6 and scss are now usable
- Our code will be in the `src` directory and compiled to `lib` directory which is not pushed to git. all scripts should be included into the manifest from the `lib` directory
- libraries will be in the external directory and included into the `manifest` directly
- install dependencies with `yarn install` or `npm install`
- compile the code with `yarn build` or `npm run build`
- compile with hot reloads with `yarn start` or `npm run start`
- clean compiled code with `yarn clean` or `npm run clean`
- create zip file for submitting to google developer dashboard with `yarn release` or `npm run release`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
+$ yarn build
+```
+Then open Google Chrome, go to `chrome://extensions/`, add the project folder using 'Load Unpacked Extenson' and you're good to go.
+Use `yarn start` instead of `yarn build` to enable hot reloading.
+NOTE: `yarn` can be replaced with `npm run`
 
 ## Contributing
-
-1. Fork it!
-2. Create your feature branch: `git checkout -b my-new-feature` off develop
```
Comment (gurpreetg) [inline README.md]: `git checkout -b feature/my-new-feature` ???

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "babel-cli": "^6.0.0"
+  },
+  "scripts": {
+    "build-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o lib/",
+    "watch-css": "npm run build-css && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o lib/ --watch --recursive",
+
+    "build-es6": "babel src/js --out-dir lib/js --source-maps -x .js --presets=es2015",
+    "watch-es6": "babel --watch src/js --out-dir lib/js --source-maps -x .js --presets=es2015",
+
+    "build": "npm-run-all -p build-es6 build-es6",
```
Comment (gurpreetg) [inline package.json]: npm-run-all provides a command called run-p which is same as `npm-run-all -p` https://github.com/mysticatea/npm-run-all/blob/master/docs/run-p.md

File: package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "start": "mkdir lib; npm-run-all -p watch-es6 watch-css",
```
Comment (gurpreetg) [inline package.json]: same as above for `npm-run-all -p`

File: package.json (hunk 2 above)
Comment (gurpreetg) [reply package.json]: @peygar ^

**Comment by gurpreetg** [comment]
Left few comments! otherwise looks great. Good job @peygar

File: src/js/content_script.js [hunk 4]
```diff
@@ -31,14 +31,13 @@ function getInformationArray(html) {
...
   var nodeFilter =  function(el, section) {
-  if (el && el.nodeName && el.nodeName.toLowerCase() === "div" &&
-      el.className && el.className === "panel panel-default" &&
-      el.innerText && el.innerText.includes(section)) {
-    return true;
-  } else {
-    return false;
-  }
-
+    if (el && el.nodeName && el.nodeName.toLowerCase() === "div" &&
```
Comment (Neil511) [inline src/js/content_script.js]: What is this? Is this supposed to be with the yarn script updates?

File: src/js/content_script.js [hunk 5]
```diff
@@ -304,12 +303,12 @@ function insertCSSLinks() {
   let titleTag = $("head").find("title");
 
   $('<link> </link>')
-  .attr({"href": chrome.extension.getURL("css/spinkit.css"), "rel": "stylesheet"})
+  .attr({"href": chrome.extension.getURL("external/css/spinkit.css"), "rel": "stylesheet"})
```
Comment (Neil511) [inline src/js/content_script.js]: should we be fetching these URLs from some JSON config file?

File: src/js/content_script.js [hunk 6]
```diff
@@ -304,12 +303,12 @@ function insertCSSLinks() {
...
   .insertAfter(titleTag);
 
   // content.css
   $('<link> </link>')
-  .attr({"href": chrome.extension.getURL("css/content.css"), "rel": "stylesheet"})
+  .attr({"href": chrome.extension.getURL("lib/css/content.css"), "rel": "stylesheet"})
```
Comment (Neil511) [inline src/js/content_script.js]: Same as above.

File: src/js/content_script.js (hunk 4 above)
Comment (peygar) [reply src/js/content_script.js]: it was just a tabbing error

File: src/js/content_script.js (hunk 5 above)
Comment (peygar) [reply src/js/content_script.js]: yes, I kept that as part of the splitting up refactor

File: README.md (hunk 1 above)
Comment (peygar) [reply README.md]: I'd like to keep it as is since it could be a bug or something, and branch names don't matter much when they're not internal

**Comment by peygar** [comment]
@Neil511 good to merge?

**Comment by gurpreetg** [comment]
Seems like Neil is busy! but we are good w/ the PR

**Pull request merged**